deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpsMessage.java

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT license. See LICENSE file in the project root for full license information.
  */
 
-package com.microsoft.azure.sdk.iot.device.transport.amqps;
```

Comment:
This class is moved from the device client package to the deps package so that both iot device and service client can use it. A lot of file changes below are just a reflection of this move. This class is not a public type that we need to worry about breaking changes to.